snippets/serial-console/README.rst

```diff
@@ -3,7 +3,14 @@
 UART Console Snippet (uart-console)
 ###################################
```

Comment:
```suggestion
UART Console Snippet (serial-console)
#####################################
```

Reply:
Related to this, should `CONFIG_UART_CONSOLE=y` be removed from this snippet? 
I'm getting a bit confused by the fact that some parts of this snippet use "serial" and others use "uart"